kratos/processes/skin_detection_process.cpp

```diff
@@ -215,8 +224,13 @@ void SkinDetectionProcess<TDim>::FillAuxiliaryModelPart(
 
     // Now we set the flag on the nodes. The list of nodes of the auxiliar model part
     auto& r_nodes_array = rAuxiliaryModelPart.Nodes();
-
     VariableUtils().SetFlag(INTERFACE, true, r_nodes_array);
+
+    // In case we are in MPI we syncronize the INTERFACE flag
+    if (mrModelPart.IsDistributed()) {
+        auto& r_communicator = mrModelPart.GetCommunicator();
+        r_communicator.SynchronizeOrNodalFlags(INTERFACE);
+    }
```

Comment:
```suggestion
     mrModelPart.GetCommunicator().SynchronizeOrNodalFlags(INTERFACE);
```

if possible avoid this check as it is cluttering the code
it is designed to work in both cases